docs/rpc/hardware-rpc.md

```diff
@@ -6,47 +6,75 @@ sidebar_position: 1
 description: NEAR RPC Node Hardware Requirements
 ---
 
-This page covers the minimum and recommended hardware requirements for engaging with the NEAR platform as a RPC node.
+This page covers the minimum and recommended hardware requirements for engaging with the NEAR platform as an RPC node.
 
 For testing your RPC once the node is fully sync'd, see [this example RPC request.](https://docs.near.org/api/rpc/network#node-status)
 
+# Mainnet
+
 ## Recommended Hardware Specifications {#recommended-hardware-specifications}
 
-| Hardware       | Recommended Specifications                                               |
-| -------------- |--------------------------------------------------------------------------|
-| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent                           |
-| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI                          |
-| RAM            | 20GB DDR4                                                                 |
-| Storage        | 1TB SSD (NVMe SSD is recommended. HDD will be enough for localnet only ) |
+| Hardware       | Recommended Specifications                      |
+| -------------- |-------------------------------------------------|
+| CPU            | 8-Core (16-Thread) Intel i7/Xeon or equivalent  |
+| CPU Features   | CMPXCHG16B, POPCNT, SSE4.1, SSE4.2, AVX, SHA-NI |
+| RAM            | 32GB DDR4                                       |
+| Storage        | 1TB NVMe SSD                                    |
```

Comment:
2.4TB disk for RPC

Reply:
We could also add a Note explaining that rpc/hot storage is based on gc_num_epochs_to_keep = 5.
Decreasing this means less disk required, increasing - more data stored and higher disk usage.

Reply:
Min gc_num_epochs_to_keep is 3, right?
Do you know how much ssd is required with gc_num_epochs_to_keep = 3?

Reply:
Don't know about 3 epochs as we run all nodes with 5.
Don't think this is a critical piece of information but rather a baseline for these numbers.